src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -80,6 +80,19 @@ namespace cryptonote
 #define CORE_RPC_STATUS_NOT_MINING "NOT MINING"
 #define CORE_RPC_STATUS_PAYMENT_REQUIRED "PAYMENT REQUIRED"
 
+static inline const std::string get_rpc_status(const bool trusted_daemon, const std::string &s)
```

Comment:
If you drop the `static` keyword, the linker will merge multiple potential copies in the runtime binary. Otherwise, each compilation unit will have its own copy, and I don't see a good reason for this.